Src/GoogleApis.Tests/Apis/Requests/PageStreamerTest.cs

```diff
@@ -0,0 +1,241 @@
+﻿/*
+Copyright 2015 Google Inc
```

Comment:
My bad about the year in my previous comment. You can write 2015 or 2016, whatever you feel more comfortable. I would actually suggest 2016, because it's the year of commit.